.github/PULL_REQUEST_TEMPLATE/pull_request_template_berk.md

```diff
@@ -0,0 +1,85 @@
+# Guidelines
+
+## _Soft_ Schema Freeze
+
+The `nmdc-schema` and `berkeley-schema-fy24` schemas are under a soft freeze, which means changes **should not** be made that have any downstream implications. To ensure this, all PRs created creating during the freeze will be closely reviewed with **every** component of the NMDC system in mind.
+
+## Reviewers
+
+To ensure no changes are made unexpectedly, PR creators will request reviews from _all_ [Berkeley Schema Roll Out task coordinators](https://docs.google.com/document/d/1XXN1YuaBuSkxPXeiLKm5YxYzXTamBPQrzzeLhlh7PWs/edit#heading=h.u52g8v319adh).
+
+We expect task coordinators to review PRs and provide feedback/approval within 1 week of when they are identified as reviewers. 
+
+PRs will **NOT** be merged until all task coordinators (or one of their delegates) have approved. 
+
+Expedition, questions, and discussion can happen at any meeting.
+
+Delays in review & merging should be addressed in meetings or with NMDC leadership.
+
+# PR Information
+
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Optimization
+- [ ] Documentation
+- [ ] Schema change: Content
+  - `slot` or `class` name
+  - changes to a `slot`'s `multivalued` state. `true` means the schema expects a list of values and `false` means  a single value is expected.
+  - any changes to the ways `slot`s are assigned to any `class`es
+  - creating a new `slot` or `class`
+  - additions, removals or changes to the `permissible_values` in an `enum`
+- [ ] Schema change: Cleanup
+  - descriptions of `slot`s, `class`es, or `enum`s
+  - removal of unused, commented, or invalid code
+  - updated `mappings` of `slot`s, `class`es, or `slot`s to ontologies
+  - addition of a new `enum` in the absence of making that `enum` the `range` for any existing `slot`
```

Comment:
I made two groups:

- Structure and content
- Cleanup and preparation

I also standardized the phrasing of the sub-bullets.

```suggestion
- [ ] Schema change: Structure and content
  - created, updated, or deleted a `class`, `slot`, or `enum`
  - changed whether a `slot` is `multivalued`
  - changed the way a `slot` is assigned to a `class`
  - changed the `permissible_values` of an `enum`
  - _etc._
- [ ] Schema change: Cleanup and preparation
  - updated the description of a `class`, `slot`, or `enum`
  - updated the `mappings` of a `class`, `slot`, or `enum` to an ontology
  - added an `enum` for _future_ use (it is not in the `range` of any `slot`)
  - _etc._
```

Reply:
Some of the attendees of today's metadata meeting discussed this suggestion today and nobody expressed objections to it.

Also, some of them clarified that they see the first group as "more likely to have downstream implications" and the second group "less likely to have downstream implications" (relative to the other group).